scripts/release.sh

```diff
@@ -459,7 +463,7 @@ build_images() {
     # the images we already have locally.
     if [ -z "$SKIP_BUILD" ]; then
       echo "Building $image:$TAG ..."
-      $NIX_BUILD --out-link "$archive-image" -A "images.$BUILD_TYPE.$archive" --arg allInOne "$ALL_IN_ONE" --arg incremental "$INCREMENTAL" --argstr product_prefix "$PRODUCT_PREFIX"
+      $NIX_BUILD --out-link "$archive-image" -A "images.$BUILD_TYPE.$archive" --arg allInOne "$ALL_IN_ONE" --arg noDebugSymbols "$DISABLE_DEBUG_SYMBOLS" --arg incremental "$INCREMENTAL" --argstr product_prefix "$PRODUCT_PREFIX"
```

Comment:
Just wondering, does it make sense to also allow passing custom builds flags?

Reply:
Yes that would be easier. I'll rework this PR to do that.

Reply:
I've made this change, PTAL.